tests/legacy-cli/e2e/setup/002-npm-sandbox.ts

```diff
@@ -25,9 +29,22 @@ export default async function () {
     process.env['NPM_CONFIG_legacy_peer_deps'] = 'true';
   }
 
-  // Configure the registry and prefix used within the test sandbox
-  await writeFile(npmrc, `registry=${npmRegistry}\nprefix=${npmModulesPrefix}`);
+  // Configure the registry and create the prefix directories
+  await writeFile(npmrc, [`registry=${npmRegistry}`, `prefix=${npmModulesPrefix}`].join('\n'));
+  await writeFile(
+    yarnrc,
+    [
+      `registry ${npmRegistry}`,
+      `prefix ${yarnModulesPrefix}`,
```

Comment:
This is what should be on the PATH and contains the symlinks to the actual bin files.